### ISSUE TITLE
Error when clicking "save preferences" on GE/Jasco/Honeywell new style dimmers

### ISSUE THREAD

**Issue by Botched1** [jvmahon/HubitatCustom#4]
Just FYI that I get this error on all GE/Jasco/Honeywell New Style Dimmers, even though a value is specified for parameter 19:

```
[dev:139](http://192.168.2.162/logs#dev139)2020-12-30 10:12:28.329 am [warn](http://192.168.2.162/device/edit/139)Device 4-Left Dimmer: Can't set parameter 19, Incomplete parameter list supplied... syntax: setParameter(parameterNumber,size,value), received: setParameter(19, 1, null).
```

Note that no matter what is selected in the drop down for Parameter 19 it gives the same error when clicking "save preferences".

![image](https://user-images.githubusercontent.com/19985320/103390122-4ec72f00-4ad8-11eb-85c1-5a10ce794df4.png)

![image](https://user-images.githubusercontent.com/19985320/103390133-61416880-4ad8-11eb-9f86-b0a4777f0375.png)

**Comment by jvmahon** [comment]
I just posted a version 0.0.3 which I think (hope!) fixes this -- at least it fixed it for a similar error on a different device.

Let me know if it now works or doesn't.

**Comment by jvmahon** [comment]
Just posted 0.0.5 - let me know if that works for the parameter settings on your device. If so, close out the comment. If not, let me know what's happening. Thanks.

**Comment by Botched1** [comment]
Seems fixed in 0.0.5. Thanks.

**Comment by jvmahon** [comment]
1 error down. thanks.
